apps/src/templates/BatchEdit.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from 'react';
```

Comment:
I'm not sure why this file is here...?

Reply:
Sloppy `git add`ing on my part! Fixed now